velox/docs/configs.rst

```diff
@@ -118,6 +118,17 @@ Maximum amount of memory in bytes that a hash join build side can use before spi
 Maximum amount of memory in bytes that an order by can use before spilling.
 0 means unlimited.
 
+``spillable-reservation-growth-pct``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``25``
+
+The spillable memory reservation growth percentage of the previous memory
+reservation size. 25 means exponential growth along a series of integer powers
```

Comment:
> 25 means exponential growth along a series of integer powers of 5/4.

I don't quite understand what this means. Would you explain? Will other readers understand?